_blog/2024-02-09-using-clerk-in-rust.mdx

```diff
@@ -0,0 +1,1014 @@
+---
+title: "Using Clerk authentication in Rust"
+description: This article is on how to use Clerk authentication in Rust based back-end applications and little bit of an extension on integration of shared database using Shuttle. 
```

Comment:
This can be a bit shorter and more consice. It can also mention the keywords (Actix, React, etc) that did not fit into the title.

Reply:
Sure I will add those as well